File: JAVA/src/fibonacci.java
import java.util.Scanner;

public class fibonacci {
    public static void main(String... s) {
        System.out.print("Enter a postive number: ");
        Scanner myObj = new Scanner(System.in);
        int num = myObj.nextInt();
        if (num <= -1) {
            System.out.println("Please enter postive number");
        } else {
            int t0 = 0, t1 = 1;
            int temp = 0;
            for (int i = 0; i < num - 1; i++) {
                temp = t0 + t1;
                t0 = t1;
                t1 = temp;
            }
            System.out.println("Fibonacci Series of " + num + " term " + "is " + temp);
        }
    }
}

File: Javascript/fibonacci.js
let first = 0;
let second = 1;
let nxt;
let num = prompt("Enter a postive number:");
if (num <= 1) {
  document.write("Please Enter a +ve number");
} else {
  for (let i = 1; i <= num; i++) {
    document.write("<br>", first);
    nxt = first + second;
    first = second;
    second = nxt;
  }
}
